src/crate/crash/sysinfo.py

```diff
@@ -50,6 +50,19 @@ class SysInfoCommand(object):
                          where schema_name
                          not in ('information_schema', 'sys') """ ]
 
+    CLUSTER_INFO_TABLE_SCHEMA = [ """ select count(distinct(id)) as number_of_shards,
```

Comment:
probably keeping the same of queries a little bit too much, we can just change the last item in the list(modify, append, etc)